api/grpcserver/mesh_service.go

```diff
@@ -264,6 +264,14 @@ func (s MeshService) getTxIdsFromMesh(minLayer types.LayerID, addr types.Address
 	return txIDs, nil
 }
 
+func convertLayerID(l types.LayerID) *pb.LayerNumber {
```

Comment:
other changes seem to be unrelated, can we keep only this one and revert other changes

Reply:
done, the mocks aren't reverted though, I hope that's no problem